scripts/2.frames.sh

```diff
@@ -7,7 +7,21 @@ fi
 WORKING_DIR="$BASE_DIR/temp/demo_output"
 MODULE="frames"
 
+FRAME_METHOD="opencv"
+CHANGE_THRESHOLD="1e-5"
+SEQUENTIAL_FRAMES="false"
+FPS="1"
+
+CONFIG=$( jq -n \
+	--arg m "$FRAME_METHOD" \
+	--argjson ct "$CHANGE_THRESHOLD" \
+	--argjson seq "$SEQUENTIAL_FRAMES" \
+	--argjson fps "$FPS" \
+	'{method: $m, change_threshold: $ct, sequential: $seq, fps: $fps}'
+)
+
```

Comment:
My only comment would be that these parameters could be better structured/documented-- but that's not to do with this PR, it's on us c.f. #25